### DISCUSSION TITLE
undetected_chromedriver not found

### DISCUSSION THREAD

**Discussion by rodolfohro** [charlesbel/Microsoft-Rewards-Farmer#446] [Q&A]
![Screenshot 2024-02-22 122653](https://github.com/charlesbel/Microsoft-Rewards-Farmer/assets/71455792/179867c2-be0b-4462-aa14-c16c4b1f50c2)

i keep getting this error despite follow all the tutorial and make the changes suggested here on the Discussions, anyone knows why ?

**Comment by gambit820** [comment]
you need to install setuptools `pip install setuptools` after that you will get another error ( no module named 'blinker._saferef') to fix that install blinker 1.7 `pip install blinker==1.7.0`

**Reply by Mahi4039** [reply]
This thing still working
